vips/vip-498/bsctestnet-2.ts

```diff
@@ -0,0 +1,257 @@
+  };
+
+  return makeProposal(
+    [
+      // Add sUSDe Market
+      // oracle config
+      {
+        target: sepolia.REDSTONE_ORACLE,
+        signature: "setDirectPrice(address,uint256)",
+        params: [USDe, parseUnits("1", 18)],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: sepolia.CHAINLINK_ORACLE,
+        signature: "setTokenConfig((address,address,uint256))",
+        params: [[USDe, CHAINLINK_USDe_FEED, STALE_PERIOD_26H]],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: sepolia.RESILIENT_ORACLE,
+        signature: "setTokenConfig((address,address[3],bool[3]))",
+        params: [
+          [USDe, [sepolia.REDSTONE_ORACLE, sepolia.CHAINLINK_ORACLE, sepolia.CHAINLINK_ORACLE], [true, true, true]],
+        ],
+        dstChainId: LzChainId.sepolia,
+      },
+
+      // Market configurations
+      {
+        target: VsUSDe_CORE,
+        signature: "setReduceReservesBlockDelta(uint256)",
+        params: ["7200"],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: sUSDe,
+        signature: "faucet(uint256)",
+        params: [sUSDe_INITIAL_SUPPLY],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: sUSDe,
+        signature: "approve(address,uint256)",
+        params: [sepolia.POOL_REGISTRY, 0],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: sUSDe,
+        signature: "approve(address,uint256)",
+        params: [sepolia.POOL_REGISTRY, sUSDe_INITIAL_SUPPLY],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: sepolia.POOL_REGISTRY,
+        signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+        params: [
+          [
+            VsUSDe_CORE,
+            parseUnits("0.72", 18), // CF
+            parseUnits("0.75", 18), // LT
+            sUSDe_INITIAL_SUPPLY, // initial supply
+            sepolia.NORMAL_TIMELOCK,
+            parseUnits("20000000", 18), // supply cap
+            parseUnits("0", 18), // borrow cap
+          ],
+        ],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: VsUSDe_CORE,
+        signature: "transfer(address,uint256)",
+        params: [ethers.constants.AddressZero, parseUnits("100", 8)], // around $100
+        dstChainId: LzChainId.sepolia,
+      },
+      (() => {
+        const vTokensMinted = convertAmountToVTokens(USDe_INITIAL_SUPPLY, parseUnits("1", 28));
+        const vTokensRemaining = vTokensMinted.sub(parseUnits("100", 8));
+        return {
+          target: VsUSDe_CORE,
+          signature: "transfer(address,uint256)",
+          params: [sepolia.VTREASURY, vTokensRemaining],
+          dstChainId: LzChainId.sepolia,
+        };
+      })(),
+
+      {
+        target: VsUSDe_CORE,
+        signature: "setProtocolSeizeShare(uint256)",
+        params: [parseUnits("0.05", 18)],
+        dstChainId: LzChainId.sepolia,
+      },
```

Comment:
I would also update the IR of the sUSDe market here, as I am suggesting here: https://github.com/VenusProtocol/isolated-pools/pull/526/files#r2093308016